plugins/jdg72/src/main/java/org/radargun/service/JDG72MultimapCacheOperations.java

```diff
@@ -0,0 +1,83 @@
+package org.radargun.service;
+
+import java.util.Collection;
+import java.util.function.Predicate;
+
+import org.infinispan.multimap.api.EmbeddedMultimapCacheManagerFactory;
+import org.infinispan.multimap.api.MultimapCacheManager;
+import org.radargun.traits.MultimapCacheOperations;
+
+
+public class JDG72MultimapCacheOperations extends Infinispan92MultimapCacheOperations {
+
+   public JDG72MultimapCacheOperations(InfinispanEmbeddedService service) {
+      super(service);
+   }
+
+   @Override
+   public <K, V> MultimapCache<K, V> getMultimapCache(String cacheName) {
```

Comment:
Is it necessary to implement all these methods again? I think they're inherited from the Infinispan92 plugin automatically. IMO, this class should be more or less empty.

Reply:
Unfortunately, yes. I need to implement it because of class MultimapCache<K, V> has been placed in different packages. Upstream: org.infinispan.multimap.api.embedded.MultimapCache<K, V> and  downstream: org.infinispan.multimap.api.MultimapCache<K, V> and thus I can not simply extend upstream class.

Reply:
I'm not talking about extending the upstream class. You're already extending Infinispan92MultimapCacheOperations so the methods should be inherited from there.